primerpedia.js

```diff
@@ -273,12 +279,22 @@ window.onload = function () {
 			try {
 				copyShareLinkInputElement.select(); // select the contents of the input
 				document.execCommand("copy"); // copy the selection into the clipboard
+
+				toggleVisibility(notificationElement, true); // show notification to user
+				// hide notification after an agreeable time
+				setTimeout(function () {
+					toggleVisibility(notificationElement, false); //
+				}, notificationTimeoutInMs);
```

Comment:
Yes.

This is the code of the example. Most of it would need to remain as is:

```CSS

    #note {
         /* removed for brevity */
        -webkit-box-shadow: 0 0 5px black;
        -moz-box-shadow:    0 0 5px black;
        box-shadow:         0 0 5px black;
    }
    .cssanimations.csstransforms #note {
        -webkit-transform: translateY(-50px);
        -webkit-animation: slideDown 2.5s 1.0s 1 ease forwards;
        -moz-transform:    translateY(-50px);
        -moz-animation:    slideDown 2.5s 1.0s 1 ease forwards;
    }

    #close {
      /* removed for brevity */
    }
    .cssanimations.csstransforms #close {
      display: none;
    }

    @-webkit-keyframes slideDown {
        0%, 100% { -webkit-transform: translateY(-50px); }
        10%, 90% { -webkit-transform: translateY(0px); }
    }
    @-moz-keyframes slideDown {
        0%, 100% { -moz-transform: translateY(-50px); }
        10%, 90% { -moz-transform: translateY(0px); }
    }
```

Also apparently it is highly encouraged to use modernizr to ensure that the current browser actually supports keyframes (it's only a [working draft](https://developer.mozilla.org/en-US/docs/Web/CSS/%40keyframes)).

All other solutions without keyframes use two different classes to transition from `opacity:0` to `opacity:1` which would require additional code to manage those.
CSS Animations are not really my forte - maybe create a new "request for help" / "hacktoberfest" task?  Just an idea.

Reply:
Yeah, it may make sense to set up an issue to convert it to css. It's more important to have the functionality than to have its code written a particular way.

In any case, I don't think we'd need the box-shadow definitions, nor the whole slide up/down motion -- just show/hide would be enough, even if the transition is instantaneous (which is what the javascript does anyway).